build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -32,9 +32,12 @@ public class CompileTargets
             { "linux-x64", "linux-x64" },
             { "ubuntu.14.04-x64", "ubuntu.14.04-x64" },
             { "ubuntu.16.04-x64", "ubuntu.16.04-x64" },
+            { "ubuntu.16.10-x64", "ubuntu.16.10-x64" },
             { "ubuntu.14.04-arm", "ubuntu.14.04-arm" },
             { "ubuntu.16.04-arm", "ubuntu.16.04-arm" },
-            { "ubuntu.16.10-x64", "ubuntu.16.10-x64" },
+            { "ubuntu.14.04-arm64", "ubuntu.14.04-arm64" },
```

Comment:
Please comment out these 3 lines - they can only be enabled when we enable official build for Arm64.